server_manager/index.html

```diff
@@ -27,6 +27,17 @@
       http-equiv="Content-Security-Policy"
       content="default-src 'self' 'unsafe-inline' outline: data:; connect-src https: 'self'; frame-src https://s3.amazonaws.com/outline-vpn/ ss:"
     />
+
+    <style>
+      /* We declare this here to sidestep webpack */
```

Comment:
What do you mean with sidestepping Webpack? Why do you need that?

Reply:
Why are you using a font for the material icons? I'm pretty sure we already have a way to use those icons, as we use them in different places.